src/containers/Masthead/MastheadContainer.jsx

```diff
@@ -234,4 +234,7 @@ MastheadContainer.propTypes = {
   }).isRequired,
 };
 
-export default compose(withApollo, injectT)(MastheadContainer);
+export default compose(
```

Comment:
Hvorfor er tillatt linjelengde gått ned? Vil heller ha lengre max-lengde på linjer i stedet for mindre

Reply:
Nye Prettier gjør det, maks-lengden er fortsatt lik.

Reply:
Men kan vi innføre en større linjelengde isåfall? Det der syns jeg er litt latterlig kort

Reply:
Vi kan stemme over det når alle er tilbake fra konferanse/ferie 😄